packages/contracts/scripts/shared/types/handler.ts

```diff
@@ -0,0 +1,15 @@
+import { Env } from "./env"
+
+export type DeployFuncParam = {
+    env: Env
+    args: any
+};
+
+export type DeployFuncCallback = (params: DeployFuncParam) => Promise<any>;
```

Comment:
I'm surprised that `<any>` passes the linter. These should be set as the return data type, or worst case, `<unknown>`

Reply:
oh well, I didn't consider `any` type here. but we have a schema validation for arguments so we can make it either `any` or `unknown`. but I am leaning to `any`

Reply:
Yes but the linter should not allow `any` in TypeScript (kills the point of types) and I want to enforce that on the front-end when we get it handled in the deploy scripts etc.

I recommend you try using `unknown` and if it's too burdensome you can tell the linter to ignore that error in those files by adding a comment in the first line. If that's too burdensome then I would consider putting it off til another day (we have bigger fish to fry)

Reply:
got it. Let me define types for `any` type.

done in https://github.com/ubiquity/ubiquity-dollar/pull/261/commits/d560f25f1daa4454936aed6314210bb22d26ba7c